responses/_recorder.py

```diff
@@ -38,19 +39,44 @@ def _remove_nones(d: "Any") -> "Any":
 
 def _dump(
     registered: "List[BaseResponse]",
-    destination: "Union[BinaryIO, TextIOWrapper]",
+    config_file: "Union[str, os.PathLike]",
```

Comment:
that I understand. Why is this change necessary ?

Reply:
the question is not about the naming of rge arguments but more about why we need to change the existing behaviour ?

what is the benefit of doing so over the initial implementation ?

Reply:
Because in the new version, we need to save the response body to `.bin` files, which means we need to write other files besides `responses.yaml`. If you still maintain the original mode of passing in the output buffer, it is obviously impossible to do this.